UnityProject/Assets/Scripts/Shuttles/CargoShuttle.cs

```diff
@@ -133,29 +145,44 @@ public bool SpawnOrder(CargoOrder order)
 		return (true);
 	}
 
+	private void GetAvailablePositions()
+	{
+		Vector3Int pos;
+		availableSpawnSlots = new List<Vector3Int>();
+
+		for (int i = -shuttleHeight; i <= shuttleHeight; i++)
+		{
+			for (int j = -shuttleWidth; j <= shuttleWidth; j++)
+			{
+				pos = Vector3Int.RoundToInt(transform.position);
```

Comment:
ideally it would be `pos = matrixMove.State.Position.RoundToInt()`, as `transform.position` isn't guaranteed to be accurate and only represents clientside lerping position (that can theoretically be way off)
You'll need to cache a reference to MatrixMove somewhere like OnEnable() though (or do lazy init with backing field), and it would be even safer if you placed `[RequireComponent(typeof(MatrixMove))]` attribute above CargoShuttle class definition